codegen.json

```diff
@@ -12,6 +12,17 @@
       "schema": "packages/server/graphql/nestedSchema/GitLab/gitlabSchema.graphql",
       "documents": "packages/server/graphql/nestedSchema/GitLab/queries/*.graphql",
       "plugins": ["typescript", "typescript-operations", "add"]
+    },
+    "packages/server/graphql/intranetSchema/sdl/resolverTypes.ts": {
+      "schema": "packages/server/graphql/intranetSchema/intranetSchema.graphql",
+      "plugins": ["typescript", "typescript-resolvers", "add"],
+      "config": {
+        "contextType": "../../graphql#InternalContext",
+        "mappers": {
```

Comment:
+1 We might want to set `"defaultMapper": void` or similar to avoid the case where people don't define it.
We could even define `export type No_model_added_in_codegen_json = {}` and set it as default mapper, just so developers are pointed to it more directly.

Reply:
i like the idea of forcing an explicit mapper!
i'll play around with the subsequent PR to test `void` and `never` and see which one works best.